mungegithub/mungers/submit-queue.go

```diff
@@ -1074,18 +1082,76 @@ func (sq *SubmitQueue) cleanupOldE2E(obj *github.MungeObject, reason string) {
 
 }
 
-func priority(obj *github.MungeObject) int {
-	// jump to the front of the queue if you don't need retested
-	if obj.HasLabel(retestNotRequiredLabel) || obj.HasLabel(retestNotRequiredDocsOnlyLabel) {
-		return retestNotRequiredMergePriority
+const (
+	criticalFixPrio = iota
+	retestNotRequiredPrio
+	retestNotRequiredDocsOnlyPrio
+	multirebasePrio
+	fixPrio
+	blocksOthersPrio
+	lowPrio
+
+	largestHighPriority = retestNotRequiredDocsOnlyPrio
+	largestLowPriority  = blocksOthersPrio
+
+	LESS = iota
```

Comment:
it would be more idiomatic to use -1, 1, 0 instead of LESS, MORE, EQUAL constants